week-6/Homework/mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,64 @@
 // Write your code here
+let images = [
+    { id: "01", 
+      src: "images/01.jpg" },
+    {
+      id: "02",
+      src: "images/02.jpg",
+    },
+    {
+      id: "03",
+      src: "images/03.jpg",
+    },
+    {
+      id: "04",
+      src: "images/04.jpg",
+    },
+  ];
```

Comment:
Very nice. 

Note: as a general practice, we tend to use numbers for `id` instead of a string.